doc/distrib/NodeHelpFiles/_review/Revit.Elements.Room.FinishBoundary.md

```diff
@@ -0,0 +1,10 @@
+## In Depth
+`Room.FinishBoundary` returns a nested list representing the given room's finish boundary. In the returned list, the first sublist represents the outermost curves, while subsequent lists represent loops within the room. Finish boundaries occur on Revit rooms on the wall's section that is defined as finish.
```

Comment:
Similar to previous ones, suggest rephrasing the last sentence as "The room boundary returned by this node is located at the finish face inside the Revit room". Along with a link to the documentation:https://help.autodesk.com/view/RVT/2024/ENU/?guid=GUID-0BB62832-36DD-4E06-A9D4-EE98CE0FCF89